docs/source/guides/using_woodwork_with_dask_and_koalas.ipynb

```diff
@@ -164,32 +164,24 @@
     "\n",
-    "When a Woodwork DataTable is created, the dtype of the underlying dataframe columns are converted to a set of standard dtypes, defined by the LogicalType `pandas_dtype` property. Woodwork uses several nullable dtypes such as `Int64`, `string` and `boolean` which are not compatible with Koalas. In addition the `category` dtype cannot be used with Koalas DataFrames.\n",
+    "When Woodwork is initialized, the dtype of the DataFrame columns are converted to a set of standard dtypes, defined by the LogicalType `primary_dtype` property. By default, Woodwork uses the `category` dtype for any categorical logical types, but this is not available with Koalas.\n",
     "\n",
-    "For LogicalTypes that have `pandas_dtype` properties that are not compatible with Koalas, Woodwork will try to convert the column dtype, but will be unsuccessful. At that point, Woodwork will use a backup dtype that is compatible with Koalas. The implication of this is that creating a DataTable from a Koalas DataFrame may result in dtype values that are different than the values you would get when creating the DataTable from an identical pandas DataFrame.\n",
+    "For LogicalTypes that have `primary_dtype` properties that are not compatible with Koalas, Woodwork will try to convert the column dtype, but will be unsuccessful. At that point, Woodwork will use a backup dtype that is compatible with Koalas. The implication of this is that using Woodwork with a Koalas DataFrame may result in dtype values that are different than the values you would get when working with an otherwise identical pandas DataFrame.\n",
     "\n",
-    "The backup datatypes used are as follows:\n",
-    "`Int64`: `int64`\n",
-    "`boolean`: `bool`\n",
-    "`string`: `object`\n",
-    "`category`: `object`\n",
-    "\n",
-    "Since Koalas does not support the `category` dtype, any column that is inferred or specified with a logical type of `Categorical` will have its values converted to strings and stored with a dtype of `object`. This means that a categorical column containing numeric values, will be converted into the equivalent string values when the DataTable is created.\n",
-    "\n",
-    "As Koalas does not support the nullable `boolean` dtype, Woodwook will fall back to use the compatible `bool` dtype for columns with a `Boolean` LogicalType. If the corresponding column contains any missing values, these missing values will be converted to `False` values when the column dtype is updated.\n",
+    "Since Koalas does not support the `category` dtype, any column that is inferred or specified with a logical type of `Categorical` will have its values converted to strings and stored with a dtype of `string`. This means that a categorical column containing numeric values, will be converted into the equivalent string values.\n",
```

Comment:
DataTable in the sentence below this one

Reply:
Fixed